### ISSUE TITLE
Update Back Stack activities after changing locale

### ISSUE THREAD

**Issue by Druux** [greenrobot/EventBus#233]
After researching for like hours I decided to try greenrobot's EventBus. What I basically want to do is to update the language of my app WITHIN my app (not device-wide). Is there any possibility to create a general onEvent() for each of my activities after the locale change has been posted?

**Comment by knezmilos13** [comment]
Hey, I've also started using event bus recently, and also had a requirement to change the language of the application from within. It didn't occur to me to use the event bus for that, though. Did you manage anything with that approach? I did it, in my case, by sending the user back to the main activity after language change, with all of my activities extending an abstract activity that forces the system to switch the language in onCreate.

Your approach, in theory, sounds much more responsive, although it seems it would require much more work. Every activity of yours would need to be attached to the event bus, and listen for an appropriate event and then reload all strings and other resources. Activities would still have to check current language on startup since a bunch of them might not be on stack & listening to the bus when you switch your language.
